### ISSUE TITLE
Neural Network Introduction (3)

### ISSUE THREAD

**Issue by noahlefebvre** [Noahhobe/TeamOne2020#25]
https://www.youtube.com/watch?v=IHZwWFHWa-w&list=PLZHQObOWTQDNU6R1_67000Dx_ZCJB-3pi&index=2

This is part two of the video series I posted earlier. This goes deep into the mathematics of how neural networks "learn". It is hard to grasp without lots of calculus knowledge. Talks a lot about gradient descent which I do not understand well.
